granCurrent.tsv

```diff
@@ -0,0 +1,34 @@
+Package	Version
```

Comment:
No, it should be using the updatedBuildTags file (or whatever it should be called) file that is associated with the GRAN build files.